eth-contracts/migrations/11_eth_rewards_manager_migration.js

```diff
@@ -0,0 +1,46 @@
+const contractConfig = require('../contract-config.js')
+const _lib = require('../utils/lib')
+const AudiusAdminUpgradeabilityProxy = artifacts.require('AudiusAdminUpgradeabilityProxy')
+const EthRewardsManager = artifacts.require('EthRewardsManager')
+const Governance = artifacts.require('Governance')
+const MockWormhole = artifacts.require('MockWormhole') // TODO: remove this
+
+const ethRewardsManagerProxyKey = web3.utils.utf8ToHex('EthRewardsManagerProxy')
+
+module.exports = (deployer, network, accounts) => {
+  deployer.then(async () => {
+    const config = contractConfig[network]
+    const proxyAdminAddress = config.proxyAdminAddress || accounts[10]
+    const proxyDeployerAddress = config.proxyDeployerAddress || accounts[11]
+    const guardianAddress = config.guardianAddress || proxyDeployerAddress
+    // const wormholeAddress = config.wormholeAddress
+
+    const tokenAddress = process.env.tokenAddress
+    const governanceAddress = process.env.governanceAddress
+
+    const governance = await Governance.at(governanceAddress)
+
+    // Deploy EthRewardsManager logic and proxy contracts + register proxy
+    const mockWormhole = await deployer.deploy(MockWormhole, { from: proxyDeployerAddress }) // TODO: remove this
```

Comment:
mockWormhole should only be deployed if there is no legitimately configured wormholeAddress